Content.Server/Materials/MaterialStorageSystem.cs

```diff
@@ -175,6 +175,10 @@ public List<EntityUid> SpawnMultipleFromMaterial(int amount, MaterialPrototype m
         var materialPerStack = composition.MaterialComposition[materialProto.ID];
         var amountToSpawn = amount / materialPerStack;
         overflowMaterial = amount - amountToSpawn * materialPerStack;
+
+        if (amountToSpawn == 0)
+            return [];
```

Comment:
```suggestion
            return new List<EntityUid>();
```
I didn't even know making a new empty list using `[]` is a thing in C#.
Anyways, doing it the same way as above is slightly more readable imo.